README.rst

```diff
@@ -949,6 +949,28 @@ domain, you can use this option to change the base of the url, this will
 enable you to point all the API urls to your openwisp-monitoring API server's
 domain, example: ``https://mymonitoring.myapp.com``.
 
+``OPENWISP_MONITORING_INFLUXDB_MAX_RETRIES``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
let's rename these settings to be generic, so that can be reused with any other future timeseries backend we may have, eg: ``OPENWISP_MONITORING_TIMESERIES_MAX_RETRIES``